local.conf.appendix

```diff
@@ -1,3 +1,20 @@
+# Enable testing
 
+# Needed since a bbappend in meta-ivi adds libusb PACKAGECONFIG which is 
+# broken for some weird reason
+BBMASK .= "|meta-ivi/recipes-devtools/qemu/"
+
```

Comment:
This and the rest of the added lines I will keep only in the test-scripts directory.